cloud/keystone_test.go

```diff
@@ -49,6 +49,43 @@ var _ = Describe("Keystone client", func() {
 		server.Close()
 	})
 
+	Describe("Tenant ID validation", func() {
+
+		setupKeystone := func() {
+			server.AppendHandlers(
+				ghttp.RespondWithJSONEncoded(201, getV3TokensScopedToTenantResponse()),
+				ghttp.RespondWithJSONEncoded(200, getV3TenantsResponse()),
+			)
+			setupV3Client()
+		}
+
+		BeforeEach(func() {
+		})
+
+		It("Returns false when tenant id is not known", func() {
+			setupKeystone()
+
+			valid, err := client.ValidateTenantID("some-random-id")
```

Comment:
nit: there is a short cut for that (in case error should be nil):
```go
Expect(client.ValidateTenantID("some-random-id")).To(BeFalse()))
```

Reply:
I prefer explicit check